### PR TITLE
Add the capability to add to the word list rather than replace it.

### PR DESCRIPTION
**Pull request by bakert**
I want to be able to censor words specific to my context while not losing the base list. This PR adds that along with a test to make sure it's working correctly.

### CODE REVIEW

File: better_profanity/profanity.py [hunk 1]
```diff
@@ -67,6 +67,12 @@ def load_censor_words(custom_words=None):
     # The default wordlist takes ~5MB+ of memory
     CENSOR_WORDSET = all_censor_words
 
+def add_censor_words(custom_words=None):
```
Comment (snguyenthanh) [inline better_profanity/profanity.py]: I think the argument `custom_words` should not have a default value.

It doesn't make much sense to me to have a `None` argument and an unnecessary function call.

File: better_profanity/profanity.py [hunk 2]
```diff
@@ -67,6 +67,12 @@ def load_censor_words(custom_words=None):
...
+    global CENSOR_WORDSET
+
+    if custom_words:
```
Comment (snguyenthanh) [inline better_profanity/profanity.py]: Should we change this `if` block to this:
```
if not isinstance(custom_words, (list, tuple, set)):
    raise TypeError("Function 'add_censor_words' only accepts list, tuple or set.")

CENSOR_WORDSET.update(custom_words)
```

File: tests.py [hunk 3]
```diff
@@ -133,6 +133,12 @@ def test_unicode_censorship_5(self):
 
         self.assertEqual(profanity.censor(bad_text), censored_text)
 
+    def test_custom_words(self):
```
Comment (snguyenthanh) [inline tests.py]: Can you please also add another test case, to check if the default wordlist is not removed after calling `add_censor_words` ?

**Comment by snguyenthanh** [comment]
Thank you very much for your help @bakert 
I will push this PR as a new version in 1-2 days.

**Comment by snguyenthanh** [comment]
> Thank you very much for your help @bakert
> I will push this PR as a new version in 1-2 days.

A new version of v0.5.0 has been published.

**Comment by bakert** [comment]
Awesome! Thanks Son!

**Pull request merged**